lib/matplotlib/figure.py

```diff
@@ -32,7 +32,7 @@
 
 from matplotlib.cbook import Stack, iterable
 
-from matplotlib import _image
+from matplotlib import image as mimage
```

Comment:
> " lib/matplotlib/tests/baseline_images/test_backend_svg/noscale.png" doesn't look right to me.

The change is due to the fact that the backend-specific `svg.image-noscale` rcParam was removed in favor of the backend-agnostic `interpolation='none'` feature and its default in rcParam `image.interpolation`.  Therefore, in the test, all three backends exhibit the `interpolation == 'none'` behavior rather than just SVG.  The svg-specific feature was removed because (a) it would require significant additional effort to maintain and (b) it's better to be backend-generic IMHO.  I think the SVG-specific feature predates the support for the feature in all relevant backends.